src/main/java/me/zeroeightsix/kami/module/modules/misc/AutoObsidian.kt

```diff
@@ -473,13 +475,41 @@ object AutoObsidian : Module() {
         }
     }
 
+    /**
+     * @return True if there is a non-sliktouch pick in the hotbar, else returns false
+     */
+    private fun hotbarHasNonSilkTouchPick(): Boolean {
+        if (InventoryUtils.getSlotsHotbar(ItemID.DIAMOND_PICKAXE.id) == null) return false
+        for (pos in InventoryUtils.getSlotsHotbar(ItemID.DIAMOND_PICKAXE.id)!!) {
```

Comment:
Store this in a variable instead of calling getSlotsHotbar() and assert non null

Reply:
done. thanks for the suggestions.